t/web/term_table/phillipines.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../../app'
+
+describe 'Phillipines' do
+  subject { Nokogiri::HTML(last_response.body) }
+  let(:dina)     { subject.css('#mem-c5c42ac6-4942-4c32-baf7-3dae194f83bf') }
+  let(:dimaporo) { subject.css('#mem-236420bc-a15a-4727-a98a-75f870c5641f') }
+  let(:tiangco)  { subject.css('#mem-8fd6197f-f9fa-47a8-974f-50c54f08bbdd') }
+
+  before do
+    stub_popolo('ba933fa', 'Philippines/House')
+    get '/philippines/house/term-table/16.html'
+  end
+
+  describe 'alt attribute on avatars' do
+    it 'has the person name in a normal image' do
+      img = dina.css('img.person-card__image')
+      img.attr('alt').text.must_equal 'Member headshot for Abad, Henedina R.'
+    end
+
+    it 'has the person name in a placeholder image' do
+      img = dimaporo.css('img.person-card__image')
+      img.attr('alt').text.must_equal 'Placeholder image for Dimaporo, Abdullah D.'
+    end
+
+    it 'doesnt break for names with double quotes' do
+      img = tiangco.css('img.person-card__image')
+      img.attr('alt').text.must_equal 'Member headshot for Tiangco, Tobias "Toby" M.'
```

Comment:
There are actually two different images with this text, and only one of them is getting checked here. Both of them actually get returned from this css selector, though, so it should be quite easy to check both.